src/routes/+page.svelte

```diff
@@ -17,4 +54,45 @@
     {@html opengraph_html}
 </svelte:head>
 
-<div class="flex h-full w-full items-center justify-center gap-3 md:gap-[0.75vw]"></div>
+<main class="relative">
+    <navbar class="absolute inset-x-0 md:h-[4vw] md:px-[1vw] flex items-center justify-between">
+        <a href="/"><Logo class="md:w-[2vw]" /></a>
+        <a href="/" class="absolute left-1/2 -translate-x-1/2"><CoreProject /></a>
+        <div class="flex items-center md:gap-[1vw]">
+            <a
+                href="/user/register"
+                class="btn hidden bg-surface-400 font-semibold leading-none md:flex md:rounded-[0.5vw] md:px-[1.25vw] md:py-[0.85vw] md:text-[1vw]"
+            >
+                Register
+            </a>
+            <a
+                href="/user/login"
+                class="btn rounded-[0.4rem] bg-primary-500 px-5 py-4 text-base font-semibold leading-none md:rounded-[0.5vw] md:px-[1.25vw] md:py-[0.85vw] md:text-[1vw]"
+            >
+                Login
+            </a>
+        </div>
+    </navbar>
+    <landing class="flex items-center justify-between h-screen md:px-[10vw]">
+        <content>
+            <div class="flex flex-col md:gap-[2vw] relative">
+                <div class="flex flex-col md:text-[5vw] font-bold md:leading-[4.5vw]">
+                    <!-- <span>Unlock the World of</span> -->
+                    <main-typewriter bind:this={top_typewriter_el} />
+                    <typewriter bind:this={typewriter_el} />
```

Comment:
Also make this `user-select:none`

Reply:
Okay I dont think this is good.
let them select
also now we're rendering `a` tags so noo :P